src/makevoice/MakeVoice.cpp

```diff
@@ -12,11 +12,26 @@
  * platform.
  */
 #include "stdafx.h"
+#include "WindowsRegistry.h"
 #include "..\Shared\branding.h"
 #include <AutomationTtsEngine_i.c>
 #include <direct.h>
 #include <fstream>
 
+// A default text to speech voice is chosen when first used.Vocalize some
+// text to make the system chooseand save a default from the currently
+// installed voices.Additionally test that text to speech works locally.
+#define UTTERANCE_FOR_SETTING_DEFAULT_VOICE _T("Installing voice named " AUTOMATION_VOICE_NAME)
+
+// Registry path storing voice tokens.
+#define SPCAT_VOICE_TOKENS SPCAT_VOICES _T("\\Tokens")
```

Comment:
```suggestion
#define SPCAT_VOICE_TOKENS _T(SPCAT_VOICES "\\Tokens")
```

@mzgoddard As written, this line took me a little research to understand, and I still don't know the character width of a literal like `"a" L"b"`. My confusion is certainly due to my unfamiliarity with macros, but I also think the version I'm proposing reads a little better (in addition to being consistent with the definition with precedes it).

Reply:
On second thought, I'll propose this with a follow-up patch

Reply:
Here it is: https://github.com/w3c/aria-at-automation-driver/pull/26